Makefile.am

```diff
@@ -56,6 +56,8 @@ check_PROGRAMS += \
 capn_test_SOURCES = \
 	tests/capn-test.cpp \
 	tests/capn-stream-test.cpp \
+	tests/example-test.cpp \
+	tests/example.capnp.c \
 	compiler/test.capnp.c \
 	compiler/schema-test.cpp \
 	compiler/schema.capnp.c
```

Comment:
Doing something unrelated I just came across [this fix](https://github.com/opensourcerouting/c-capnproto/blob/master/compiler/c.capnp#L24-L32) in the source. I can try and use it.